### PR TITLE
Adds use case scenario to get dealer name

### PR DESCRIPTION
**Pull request by waldemarnt**
@carolinaborim and @waldemarnt pairing

### CODE REVIEW

**Comment by andressa** [comment]
👍

**Comment by lucaslago** [comment]
👍

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 1]
```diff
@@ -39,8 +39,20 @@ The API will answer with the requested equipment<br>
...
-To access this equipment, it's necessary to make a HTTP Request in the following format: GET /equipment/{id} and when the information is returned with success, a single equipment will be available.
+To access this equipment, it's necessary to make a HTTP Request in the following format: `GET /equipment/{id}` and when the information is returned with success, a single equipment will be available.
 
 The equipment on the response contains the identification information, as the manufacturing date and description, and related information to the current status. It is possible to extract the location coordinates and the latest values of CAN variables reported back from the tracker.
 
 More details can be found on link [Get single equipment section](#get-equipment-id)
+
+## Get dealer name of an equipment
+  Given I'm a developer,<br>
+  When I do a request to Dealer API, asking for a a specific dealer<br>
```
Comment (jeffhsta) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: There is a typo in this line, "..asking for a a specific..."

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ The API will answer with the requested list<br>
 
 At this [code example](https://github.com/agco-fuse/dealership-demo/blob/master/app/scripts/services/apiservice.js#L11-L19) we are requesting the API to get a list of equipment.
 
-To access this list, it's necessary to make a HTTP Request in the following format: GET /equipment and when the information is returned with success, a list of equipment will be available.
+To access this list, it's necessary to make a HTTP Request in the following format: `GET /equipment` and when the information is returned with success, a list of equipment will be available.
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: 👍

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 3]
```diff
@@ -39,8 +39,89 @@ The API will answer with the requested equipment<br>
 
 At this [code example](https://github.com/agco-fuse/dealership-demo/blob/master/app/scripts/services/apiservice.js#L21-L29) we are requesting the API to get a specific equipment by passing a valid id as a parameter.
 
-To access this equipment, it's necessary to make a HTTP Request in the following format: GET /equipment/{id} and when the information is returned with success, a single equipment will be available.
+To access this equipment, it's necessary to make a HTTP Request in the following format: `GET /equipment/{id}` and when the information is returned with success, a single equipment will be available.
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: 👍

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 4]
```diff
@@ -39,8 +39,89 @@ The API will answer with the requested equipment<br>
...
 
 The equipment on the response contains the identification information, as the manufacturing date and description, and related information to the current status. It is possible to extract the location coordinates and the latest values of CAN variables reported back from the tracker.
 
 More details can be found on link [Get single equipment section](#get-equipment-id)
+
+## Get dealer name of an equipment
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: s/Get dealer name of an equipment/Get the dealer associated to an equipment/

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 5]
```diff
@@ -39,8 +39,89 @@ The API will answer with the requested equipment<br>
...
+  Given I'm a developer,<br>
+  When I do a request to Dealer API, asking for a specific dealer<br>
+  The dealer API will answer with the requested dealer<br>
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: Any reasons for using the Connextra style in here?
I would rather explain it in plain English, given that a lot of people out there would not be familiarized with user stories.

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 6]
```diff
@@ -39,8 +39,89 @@ The API will answer with the requested equipment<br>
...
+  **How it works?**
+
+  At this [code example](https://github.com/agco-fuse/dealership-demo/blob/master/app/scripts/services/apiservice.js#L31-L38) we are requesting the API to get a specific dealer by passing a valid id as a parameter.
+
+  To access this dealer, it's necessary to make a HTTP Request in the following format: `GET /dealer/{id}` and when the information is returned with success, a single dealer will be available.
+
+## Get available CAN variables from an equipment
+  Given I'm a developer,<br>
+  When I do a request to `Equipment API`, asking for an equipment<br>
+  The `Equipment API` will answer with the equipment including all available `CAN variables`<br>
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: Same here. Explaining it verbosely would be interesting.

File: source/includes/use_case_scenarios/_use_case_scenarios.md (hunk 5 above)
Comment (carolinaborim) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: We just followed the same pattern of the other use cases examples, to keep them all in the same format.

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 7]
```diff
@@ -16,31 +16,93 @@ There is a [documentation](https://agco-fuse.github.io/documentation/#authentica
 In order to consume our API, a valid bearer token must be set in the Authorization header of every HTTP request. There is a [code example](https://github.com/agco-fuse/dealership-demo/blob/master/index.js#L17-L30) showing how to do that.
 
 ## Get a list of equipment
-Given I'm a developer with a Bearer token,<br>
-When I do a request to Equipment API, asking for a list of equipment<br>
-The API will answer with the requested list<br>
 
-**How it works?**
+To access a list of equipment, it's necessary to make a HTTP Request to [Equipment API](https://agco-fuse.github.io/documentation/#get-equipment) in the following format: `GET /equipment` and when the information is returned with success, the list will be available.
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: We could make it easier to read if we put a . (dot) between  "GET /equipment" and "when the information".

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 8]
```diff
@@ -16,31 +16,93 @@ There is a [documentation](https://agco-fuse.github.io/documentation/#authentica
...
 
 More details can be found on link [Get equipment section](#get-equipment)
 
 ## Get a single equipment
-Given I'm a developer with a Bearer token,<br>
-When I do a request to Equipment API, asking for a single equipment<br>
-The API will answer with the requested equipment<br>
 
-**How it works?**
+  To access an equipment, it's necessary to make a HTTP Request to [Equipment API](https://agco-fuse.github.io/documentation/#get-equipment) in the following format: `GET /equipment/{id}` and when the information is returned with success, a single equipment will be available.
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: We could make it easier to read if we put a . (dot) between  "GET /equipment/{id}" and "when the information".

Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: nitpick: why do we have these two empty spaces at the beginning of the paragraph?

File: source/includes/use_case_scenarios/_use_case_scenarios.md [hunk 9]
```diff
@@ -16,31 +16,93 @@ There is a [documentation](https://agco-fuse.github.io/documentation/#authentica
...
 
-To access this equipment, it's necessary to make a HTTP Request in the following format: GET /equipment/{id} and when the information is returned with success, a single equipment will be available.
-
 The equipment on the response contains the identification information, as the manufacturing date and description, and related information to the current status. It is possible to extract the location coordinates and the latest values of CAN variables reported back from the tracker.
 
 More details can be found on link [Get single equipment section](#get-equipment-id)
+
+## Get the dealer associated to an equipment
+
+  To get a dealer associated to an equipment, it's necessary to make a HTTP Request to Dealer API in the following format: `GET /dealer/{id}` and when the information is returned with success, a single dealer will be available.
```
Comment (glauco) [inline source/includes/use_case_scenarios/_use_case_scenarios.md]: We could make it easier to read if we put a . (dot) between "GET /dealer/{id}" and "when the information".

**Comment by glauco** [comment]
🐑 🇮🇹

**Comment by jeffhsta** [comment]
:+1:
